data/dellaji.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Della Ji",
+    "type": "personal",
+    "bio": "Dedicated full stack developer with expertise in devops",
+    "tags": [
+        "Open Source",
+        "Python",
+        "JavaScript",
+        "Typescript",
+        "Vue",
+        "React",
+        "Openstack"
+      ],
+    "links": [
+        {
+            "name": "Follow me on Github",
+            "url": "https://github.com/JMD110",
+            "icon": "FaGithub"
+        },
+        {
+            "name": "Follow me on Linkedin",
+            "url": "https://www.linkedin.com/in/della-ji-ba6519110/",
```

Comment:
Can you please verify the Linkedin URL?
I got a 404 when I tried to visit the page.

Thanks :)

Reply:
Sorry for the confusion. Please try again. [linkin](https://www.linkedin.com/in/della-ji-ba6519110)

Reply:
@JMD110 I don't think that your LinkedIn link is working still, but as this is not a blocker I will merge your Pull Request. 

If you want to make any changes or additions to your Profile, then please raise another PR.  I would suggest you might want to take a look at the Editing Guide in Docs when you do: https://linkfree.io/docs/how-to-guides/editing